src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs

```diff
@@ -131,6 +131,30 @@ public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmVideoBlockSpli
             StringBuffer content = string.Empty;
 
             var videoToken = splitToken.Token as DfmVideoBlockToken;
+            //TODO: fix video link
+            //if youtube.com, change to youtube-nocookie.com
+            //if channel9.msdn.com, add ?nocookie=true at end
+            if (videoToken != null && !String.IsNullOrWhiteSpace(videoToken.Link))
+            {
+                string link = videoToken.Link;
+
+                if (link.IndexOf("channel9.msdn.com", StringComparison.InvariantCultureIgnoreCase) > -1)
```

Comment:
Use Uri to parse first? it can hit such case "channel9.msdn.com.anotherwebsite.com"

Reply:
Updated to use URI